components/has/overlays/development/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base/argocd-roles
+  - https://github.com/redhat-appstudio/application-service/config/default?ref=a32c938ee1fcafaed4c64abd3ed4864121fd363c
+
+
+components:
+  - ../../components/manager-resources
+  - ../../components/enable-profiling
+
+images:
+  - name: quay.io/redhat-appstudio/application-service
+    newName: quay.io/redhat-appstudio/application-service
+    newTag: 05ee33115e4cd7ebfad61d96ffe5ef36c8159dab
```

Comment:
This doesn't match the commit SHA up above (a32c938ee1fcafaed4c64abd3ed4864121fd363c)

Reply:
yeah, WIP.  Had to manually patch it but I'm waiting for another rebase